python/src/trezorlib/cli/cardano.py

```diff
@@ -83,10 +91,26 @@ def sign_tx(client, file, protocol_magic, network_id, testnet):
         auxiliary_data,
     )
 
-    return {
-        "tx_hash": signed_transaction.tx_hash.hex(),
-        "serialized_tx": signed_transaction.serialized_tx.hex(),
-    }
+    sign_tx_response["tx_hash"] = sign_tx_response["tx_hash"].hex()
+    sign_tx_response["witnesses"] = [
+        {
+            "type": witness["type"],
+            "pub_key": witness["pub_key"].hex(),
+            "signature": witness["signature"].hex(),
+            "chain_code": witness["chain_code"].hex()
+            if witness["chain_code"] is not None
+            else None,
+        }
+        for witness in sign_tx_response["witnesses"]
+    ]
+    if auxiliary_data_supplement := sign_tx_response.get("auxiliary_data_supplement"):
```

Comment:
please don't use the walrus in trezorlib, we're still keeping python 3.6 compatibility.

(it's ok to use it in the firmware itself)

Reply:
Ok, I've removed it.